geopolars/src/geoseries.rs

```diff
@@ -550,6 +686,139 @@ mod tests {
         assert_eq!(result, correct, "Should get the correct convex hull");
     }
 
+    #[test]
+    fn skew() {
+        let geo_series = Series::from_geom_vec(&[Geometry::Polygon(polygon!(
```

Comment:
Not that familiar with these macros, but does `polygon!` not create a `Geometry::Polygon` object already? You still have to wrap the output of `polygon!` with `Geometry::Polygon`?

Reply:
Creates a geo-types polygon, not a Geometry::Polygon as far as I can tell. Switched these up to using an .into conversion though rather than wrapping them. Looks a little cleaner

Reply:
Ah I see; I forgot there was a difference